### PR TITLE
Hypershift 4.19 custom config bumps

### PR DESCRIPTION
**Pull request by Patryk-Stefanski**

### CODE REVIEW

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse pull-ci-openshift-hypershift-release-4.18-mce-images

File: ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-release-4.18.yaml [hunk 1]
```diff
@@ -50,32 +50,21 @@ releases:
...
-    candidate:
-      product: ocp
-      stream: ci
-      version: "4.18"
   n1minor:
     candidate:
       product: ocp
       stream: ci
-      version: "4.18"
+      version: "4.17"
```
Comment (bryan-cox) [inline ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-release-4.18.yaml:62]: Were these values just wrong before? Trying to understand why the value went down here and right below.

File: ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml [hunk 2]
```diff
@@ -73,32 +73,32 @@ releases:
       product: ocp
       stream: ci
       version: "4.19"
```
Comment (bryan-cox) [inline ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml:75]: Does this not go to 4.20 now?

File: ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml [hunk 3]
```diff
@@ -73,32 +73,32 @@ releases:
...
-  initial-417:
+  initial-418:
     candidate:
       product: ocp
       relative: 1
       stream: ci
-      version: "4.19"
+      version: "4.18"
```
Comment (bryan-cox) [inline ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml:81]: Was the value here just wrong before?

File: ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml [hunk 4]
```diff
@@ -73,32 +73,32 @@ releases:
...
       product: ocp
       stream: ci
-      version: "4.19"
+      version: "4.18"
   n1minor:
     candidate:
       product: ocp
       stream: ci
-      version: "4.19"
+      version: "4.18"
```
Comment (bryan-cox) [inline ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml:96]: n1minor is with respect to the latest right?

File: ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-release-4.19.yaml [hunk 5]
```diff
@@ -139,15 +139,15 @@ tests:
...
   run_if_changed: ^(test/e2e|hypershift-operator)
   steps:
     cluster_profile: hypershift
     dependencies:
-      OCP_IMAGE_LATEST: release:latest-417
-      OCP_IMAGE_N1: release:latest-417
-      OCP_IMAGE_N2: release:latest-417
-      OCP_IMAGE_PREVIOUS: release:initial-417
+      OCP_IMAGE_LATEST: release:latest-418
+      OCP_IMAGE_N1: release:latest-418
```
Comment (bryan-cox) [inline ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-release-4.19.yaml:148]: why isn't this n1minor? Similar comment for N2.

**Review by bryan-cox — COMMENTED** [review]
Several questions that might apply thoughout. I'm just confused on the numbering.

File: ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml (hunk 2 above)
Comment (bryan-cox) [reply ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml:75]: Did a quick huddle with Patryk. This was bumped previously to 4.19 when it should have been 4.18.

File: ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml (hunk 3 above)
Comment (bryan-cox) [reply ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml:81]: Did a quick huddle with Patryk. The value was wrong before.

File: ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-release-4.18.yaml (hunk 1 above)
Comment (bryan-cox) [reply ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-release-4.18.yaml:62]: Did a quick huddle with Patryk. These values were wrong before.

File: ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-release-4.19.yaml (hunk 5 above)
Comment (bryan-cox) [reply ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-release-4.19.yaml:148]: Did a quick huddle with Patryk. It gets replaced later in the test.

**Pull request merged**